packages/react-ui/lib/client.ts

```diff
@@ -0,0 +1,17 @@
+export const isBrowser = typeof window !== 'undefined';
+export const canUseDOM = isBrowser && window.document && window.document.createElement;
+
+// NOTE Some checks are used from https://github.com/arasatasaygin/is.js
+const platform = isBrowser ? ((navigator && navigator.platform) || '').toLowerCase() : '';
```

Comment:
Можно еще чуть проще.

```suggestion
const platform = ((isBrowser && navigator && navigator.platform) || '').toLowerCase();
```

Reply:
Сделал